services/event_service_test.go

```diff
@@ -486,6 +486,74 @@ func TestEventService_GetAppEvent(t *testing.T) {
 	}
 }
 
+func TestEventService_ReplayAppEvent(t *testing.T) {
+	ctx := context.Background()
+	type args struct {
+		ctx   context.Context
+		event *datastore.Event
+		g     *datastore.Group
+	}
+	tests := []struct {
+		name        string
+		args        args
+		dbFn        func(es *EventService)
+		wantErr     bool
+		wantErrCode int
+		wantErrMsg  string
+	}{
+		{
```

Comment:
This test case doesn't specify `wantErr` as `false`.

Reply:
fixed